codegen/method.go

```diff
@@ -63,10 +63,8 @@ type MethodSpec struct {
 	// Statements for reading query parameters.
 	QueryParamGoStatements []string
 
-	// ReqHeaderFields is a map of "header name" to
-	// a golang field accessor expression like ".Foo.Bar"
-	// Use to place request headers in the body
-	ReqHeaderFields map[string]HeaderFieldInfo
+	// Statements for reading request headers
+	ReqHeaderGoStatements []string
 
 	// ResHeaderFields is a map of header name to a golang
```

Comment:
The `ResHeaderField` currently does not generate any code. So there are no go statements to generate.